docs/coordinates/representations.rst

```diff
@@ -685,3 +693,22 @@ In pseudo-code, this means that a class will look like::
 
     class MyDifferential(BaseDifferential):
         base_representation = MyRepresentation
+
+.. _astropy-coordinates-create-geodetic:
+
+Creating Your Own Geodetic Representation
+-----------------------------------------
+
+Geodetic representations are implemented to allow the representation of planetary bodies.
+Your planetary class must inherit from the `~astropy.coordinates.BaseGeodeticRepresentation`.
+The equatorial radius and flattening must be both assigned via the attributes
+`_equatorial_radius` and `_flattening`. Alternatively an `_ellipsoid` attribute may be
+set to the relevant ERFA index (as passed in to `erfa.eform`).
+
+For example the spheroid describing Mars as in the
+`1979 IAU standard <https://doi.org/10.1007/BF01229508>`_ should be defined like::
+
+    class IAUMARS1979GeodeticRepresentation(BaseGeodeticRepresentation):
+
+        _equatorial_radius = 3393400.0 * u.m
+        _flattening = 0.518650e-2 * u.dimensionless_unscaled
```

Comment:
Having to add `* u.dimensionless_unscaled` to attribute is not really user friendly in my opinion.
What about a check on `_flattening` adding the units when it is missing?

Reply:
It was a bug then.
I have modified the tests and the code, otherwise when a float was passed the conversion failed.